.github/workflows/cluster_endtoend_vtorc_8.0.yml

```diff
@@ -77,6 +77,9 @@ jobs:
     - name: Run cluster endtoend test
       timeout-minutes: 30
       run: |
+        # We set the VTDATAROOT to the /tmp folder to reduce the file path of mysql.sock file
+        # which musn't be more than 107 characters long.
+        export VTDATAROOT="/tmp/"
```

Comment:
This change is unrelated to this PR, but running `make generate_ci_workflows` caused this. This probably occurred because 2 PRs got merged one which added VTOr 8.0 tests and one which set VTDATAROOT to /tmp/